misc/specs/atrium-specs-common/src/main/kotlin/ch/tutteli/atrium/specs/integration/MapEntryAssertionsSpec.kt

```diff
@@ -92,4 +103,138 @@ abstract class MapEntryAssertionsSpec(
             }
         }
     }
+
+    val keyName = "key"
```

Comment:
rename MapEntryFeatureAssertionsSpec to MapEntryAssertionsSpec and re-use here via
```
include(object: MapEntryAssertionsSpec(...){})
```
instead of duplicating

Reply:
I don't like the duplication too, but note that `MapEntryAssertionsSpec` already exists, so simply renaming and including this way is not possible, or most probably there's something that I don't quite get in your comment :(

Reply:
Yes, I guess a misunderstanding and somehow I copied the wrong class names :laughing: sorry about that. I meant rename KeyValueLikeFeatureAssertionSpec to KeyValueLikeAssertionSpec and include it here. The include should happen exactly here. All other changes in this PR are good.
See above in this class, we also have:
```
include(object : SubjectLessSpec<Map.Entry<String, Int>>(
        describePrefix,
        isKeyValue.forSubjectLess("key", 1)
    ) {})
```
the same can be used here. See the original MapEntryFeatureAssertionSpec it extended KeyValueLikeFeatureAssertions. You can use the same parameters, just use object:... instead of extending.
I hope my explanation is not too confusing

Reply:
Got it, no worries :-)

Reply:
One more thing - `KeyValueLikeFeatureAssertionsSpec` is deprecated, and `MapEntryFeatureAssertionSpec` extended `KeyValueLikeAssertionsSpec` already, so there's no need to rename it - otherwise everything is clear :-) Pushed new changes :-)